_events/talks/event-015.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Bringing Imaging-Based Artificial Intelligence into Clinical Practice in NHS Radiology"
+authors:
+    - &speaker
+      name: Dr. Rizwan Malik
+      is_speaker: true
+      affiliation: 1
+    - name: Shoaib Sufi
+      affiliation: 2
+affiliations:
+    - name: Bolton NHS Foundation Trust, UK
+      index: 1
+    - name: eScience Lab, University of Manchester, UK
+      index: 2
+author: *speaker
+category: talks
+language: English
+prerequisites: "Links are included in the summary as well as this BBC article for the background of the deployment otherwise no specialise prerequisite knowledge is required
+<a href='https://www.bbc.co.uk/news/business-52483082' class='truncated'>https://www.bbc.co.uk/news/business-52483082</a>"
+date: 2020-08-10
+---
+Whilst there are over 200 Imaging/Radiology AI Companies world wide there are currently only about 50 FDA approved algorithms
+There are also significant barriers to overcome in bringing AI to the NHS landscape.
+In Bolton NHS Foundation Trust we are the first NHS Organisation to deploy the Qure.AI solution into clinical practice.
+https://qure.ai/qxr.html
+
+In this talk we will dissect some of the barriers -- real and perceptual - which has to date limited the use of AI in clinical practice in the NHS and how we overcame these based on our lived experience. Of relevance to the software engineer audience this will include some disconnects between AI in the lab and clinical deployment.
+
+With testing turnaround times of more than 24 hours clinicians used chest radiography as the primary diagnostic tool to assess and triage COVID-19 patients. The introduction of Artificial Intelligence (AI) algorithms to radiology in Bolton resulted in a much faster and more accurate patient diagnosis. It showed quickly if someone's condition was worsening, improving or staying the same. This led to a more efficient triaging and care of patients and optimised use of critical care resources.  The AI tool also mitigated challenges presented by the loss of medical staff due to illness, self-isolation or shielding.
+
+Our vision was to give clinicians 24/7 access to an accurate and consistent decision support tool to improve chest x-ray interpretation allowing them to make quicker and better-informed decisions about patient care.
+
+We carried out extensive research before partnering with Qure.ai and this contributed to the NHSx guidance.
+https://www.nhsx.nhs.uk/media/documents/A_Buyers_Checklist_for_AI_in_Health_and_Care.pdf
+
+https://www.gov.uk/guidance/assessing-if-artificial-intelligence-is-the-right-solution#assessing-if-ai-is-the-right-solution-for-your-users-needs
```

Comment:
these two links need to be formatted and linked in the text.

Reply:
I included those, please check.